### ISSUE TITLE
Small suggestion on the stone knife recipe

### ISSUE THREAD

**Issue by tsulh** [CleverRaven/Cataclysm-DDA#20924]
I suggest to remove 'rock' from the available components, leaving only 'sharp rock' and 'ceramic shard'. it doesn't seem realistic that a human can,  just in an hour, somehow split/sharpen a rock just by hammering it with another rock. Also, sharp rocks are not that hard to find.

**Comment by Wad67** [comment]
It's entirely possible to make a 'blade' in reasonable time by hitting a rock with another rock. See: Knapping
Granted, that would most likely take an hour or two itself, resulting in quite a few 'flakes' for cutting purposes. Of course not taking into account for shaping the handle and fixing the blade within. 
Mileage may vary depending on the methods used and what would be considered a passable 'blade'.

**Comment by kevingranade** [comment]
I agree this should happen, when the recipe was added there was no sharp
rock item.

**Comment by DangerNoodle** [comment]
This should not occur unless we have a distinction between knappable rocks and useless ones. Otherwise it merely replaces one form on unrealistic behavior with another.

A pebble tool in the Oldowan style is more usable than most naturally broken rocks one is likely to find, and would not even require a handle if starting from a piece large enough to leave an untouched section to hold.

The realism failure here is that making a finished small blade with handle from a rock alone in so little time is unrealistic. But an adequate uniface takes literally nothing but a piece of the right stone and a hammerstone. Even I have been able manage that, despite insufficient practice or training to work the resulting piece into a biface.

This means if anything, what should be changed is for rocks to be a mix of conchoidal-fracture rocks and rocks with less desirable fracture patterns, and for the bare minimum craftable tool to be a pebble tool. The sharp rocks give the impression that they are modeled off of naturally-broken pieces of non-conchoidal-fracture stone. A rough edge that can tear soft material if you are desperate, whereas naturally-broken pieces of chert I have encountered tend to be sharper.

**Comment by kevingranade** [comment]
The (not entirely accurate, but workable) assumption was that sharp rocks
are knappable.

If you have a more nuanced system in mind for handling knappable
(conchoidal?) rocks, feel free to propose it, but orobably not in an issue
labelled 'small suggestion'.

**Comment by DangerNoodle** [comment]
A more nuanced suggestion and discussion issue would be ideal, but my main reason for posting a somewhat extensive statement regarding relevant points was to object to this proposed small suggestion, because it is an inadequate solution. My apologies.

**Comment by kevingranade** [comment]
Please be more clear about 'inadequate'.  The status quo is, "all rocks are
knappable", the suggestion basically bifurcates rocks into "un-knappable"
(rock) and "knappable" (sharp rock).

You're saying the new state is worse than the current status quo?

**Comment by DangerNoodle** [comment]
Main reasoning is that restricting player access to rocks is primarily a realism concern, as there are few gameplay or game balance reasons to restrict player access to cutting quality given it i.

If realism is the objective, then changing the status quo would entail more than simply restricting the recipe. A plan would be roughly:

1. For simplicity, sharp rocks could be trivially renamed to flint  or chert to represent knappable versus non-knappable stones.
2. As covered in detail in my initial reply, realistically a bare-minimum cutting tool does not require a handle and cordage, but the result would not be sheathable, nor a decent weapon. The former is more of a concern than the latter, as stone knives are already poor weapons.
3. Changing the sharp rocks to knappable rocks carries further recipe and usage implications. Now they might not count as natually sharp and thus might no longer warrant a butchering quality, while also being more usable in recipes that disallow sharp rocks due to the prior assumption that sharp rocks represent all naturally broken rocks.

To summarize, a change needs to be planned out, simply removing one line in a recipe does not think out all of the implications of what the proposed change means in realistic terms.

**Comment by Chezzo** [comment]
I am unable to find an example of someone using a sharp rock to knapp. 

When I think a sharp rock, I think a rock that is somewhat like a knife already (otherwise how does it have butchering qualities?) 

Wouldn't that make the angles that you would have to strike the sharp rock too steep to be effective?

I think the OP was thinking the rock was being sharpened, which I guess would work too.

**Comment by kevingranade** [comment]
> If realism is the objective, then changing the status quo would entail more than simply restricting the recipe.

This is IMO a false dilemma, realism is never a binary state, it is always a matter of "more realism" vs "less realism", so stating that something is "more realistic, but not realistic enough" doesn't hold up.

> For simplicity, sharp rocks could be trivially renamed to flint or chert to represent knappable versus non-knappable stones.

This is the opposite of simplicity, we want to make a distinction about knappable vs non-knappable stones, not state that all our stones are specifically flint or chert, which raises other issues, like why they appear in certain areas, and suggests other properties of the stones than the ones we are modelling.

> realistically a bare-minimum cutting tool does not require a handle and cordage, but the result would not be sheathable, nor a decent weapon.

If you want better modelling of knapped stone knives, by all means make said suggestion, but you have not made a case for why that would have anything to do with this issue.

> Changing the sharp rocks to knappable rocks carries further recipe and usage implications. Now they might not count as natually sharp and thus might no longer warrant a butchering quality, while also being more usable in recipes that disallow sharp rocks due to the prior assumption that sharp rocks represent all naturally broken rocks.

It *may* carry further recipe and usage implications, very simply, we can assert that, for the time being, it does not.
You assert:
> The sharp rocks give the impression that they are modeled off of naturally-broken pieces of non-conchoidal-fracture stone.

There is nothing in the description that indicates this, "A rock with sharp edges, that can be used as a butchering tool, if nothing else is available.  Makes a passable melee weapon.".

Most importantly though, I want to reiterate my previous statement and question, which you ignored:
> The status quo is, "all rocks are knappable", the suggestion basically bifurcates rocks into "un-knappable" (rock) and "knappable" (sharp rock).

> Are you saying the new state is worse than the current status quo?

**Comment by DangerNoodle** [comment]
> Most importantly though, I want to reiterate my previous statement and question, which you ignored:

My apologies, I evidently skipped the direct answer to your question and went into the elaboration, specifically why this suggestion is not as simple as changing a single recipe, possible complications, some conjecture on my part, and offered a somewhat rushed suggestion as a more complete change

Since you then presented reasonable (and understandable) counter-arguments to those suggestions, I would have assumed that a yes or no answer was not needed. I was incorrect.

To simplify: Yes, I believe it will be worse than the status quo, because the suggestion is a single change that does not fully consider the effects it will have.

**Comment by DangerNoodle** [comment]
> This is IMO a false dilemma, realism is never a binary state, it is always a matter of "more realism" vs "less realism", so stating that something is "more realistic, but not realistic enough" doesn't hold up.

How is this an objection? I was simply stating that a single-line change will make the use of stone tools less realistic overall than the status quo. Which means you took the exact opposite interpretation of what I intended by that statement, despite using the same reasoning regarding how realism should be handled.

> This is the opposite of simplicity, we want to make a distinction about knappable vs non-knappable stones, not state that all our stones are specifically flint or chert, which raises other issues, like why they appear in certain areas, and suggests other properties of the stones than the ones we are modelling.

This would be a bad example on my part. I mean merely that the name "sharp rock" carries implications as to what the item is, and that changing the name to something implying the item is a type of knappable stone will ensure we do not have to invent new items. I am merely unable to devise a good name for such an item.

> If you want better modelling of knapped stone knives, by all means make said suggestion, but you have not made a case for why that would have anything to do with this issue.

Case is: Requiring the player to have a knappable stone, a piece of wood, and cordage for the bare minimum cutting quality is less realistic, when only the first componant is absolutely essential. It is related to this issue in that the proposed change makes access to stone cutting tools even more restricted, when a combination of this restriction plus allowing for stone unifaces will bring the end result towards more realistic. Both suggestions compliment each other.

>It may carry further recipe and usage implications, very simply, we can assert that, for the time being, it does not.

Acknowledged. I am suspicious of half-measures, but this is not a complex source change where "complete implementation later" is a gamble. This can be put off if required.

>There is nothing in the description that indicates this, "A rock with sharp edges, that can be used as a butchering tool, if nothing else is available. Makes a passable melee weapon.".

Reasoning was based off in-game properties, not description. As iterated earlier: A natural broken piece of chert, flint, or other suitable stone would be more likely to have an edge that, in game terms, would equal a cutting quality, which the sharp rock lacks. In contrast, a naturally broken piece of non-conchoidial stone tends to have a much rougher edge, and would be more of a tearing tool at best.

**Comment by Hatfish** [comment]
Knapper here, making a rock blade can take all of one minute depending on what stones you're using and the quality you're aiming for.
If you're going for a real nice smoothed blade then yes, it will take some time to grind down the edges, but it's really not that hard to sharpen a piece of rock quickly.

**Comment by Treah** [comment]
I agree with Hatfish its not hard to get a passable cutting tool from a rock in only a few min by smashing one into the other. If nothing else I hate finding sharp rocks in the game as they re pretty much useless for other recipes such as the kiln and forge which require lots of normal rocks to build.

**Comment by Night-Pryanik** [comment]
Closing as no discussion for more than half a year.
